api_tests/e2e/rfc003/btc_eth-erc20/refund_beta.ts

```diff
@@ -0,0 +1,171 @@
+import * as bitcoin from "../../../lib/bitcoin";
+import * as chai from "chai";
+import * as ethereum from "../../../lib/ethereum";
+import { Actor } from "../../../lib/actor";
+import { ActionKind, SwapRequest, SwapResponse } from "../../../lib/comit";
+import { Wallet } from "../../../lib/wallet";
+import { BN, toBN, toWei } from "web3-utils";
+import { HarnessGlobal, sleep } from "../../../lib/util";
+import { createTests } from "../../test_creator";
+import chaiHttp = require("chai-http");
+
+const should = chai.should();
+chai.use(chaiHttp);
+
+declare var global: HarnessGlobal;
+
+(async function() {
+    const tobyWallet = new Wallet("toby", {
+        ethConfig: global.ledgers_config.ethereum,
+    });
+
+    const tobyInitialEth = "10";
+    const bobInitialEth = "5";
+    const bobInitialErc20 = toBN(toWei("10000", "ether"));
+
+    const alice = new Actor("alice", global.config, global.test_root, {
+        ethConfig: global.ledgers_config.ethereum,
+        btcConfig: global.ledgers_config.bitcoin,
+    });
+    const bob = new Actor("bob", global.config, global.test_root, {
+        ethConfig: global.ledgers_config.ethereum,
+        btcConfig: global.ledgers_config.bitcoin,
+    });
+
+    const aliceFinalAddress = "0x00a329c0648769a73afac7f9381e08fb43dbea72";
+    const bobFinalAddress =
+        "bcrt1qs2aderg3whgu0m8uadn6dwxjf7j3wx97kk2qqtrum89pmfcxknhsf89pj0";
+    const bobComitNodeAddress = bob.comitNodeConfig.comit.comit_listen;
+
+    const alphaAssetQuantity = 100000000;
+    const betaAssetQuantity = toBN(toWei("5000", "ether"));
+    const alphaMaxFee = 5000; // Max 5000 satoshis fee
+
+    const alphaExpiry: number =
+        new Date("2080-06-11T13:00:00Z").getTime() / 1000;
```

Comment:
:question: Sorry to get to this so late, but why do we only refund beta in some of these end to end tests?

Reply:
Just thinking about this now, wouldn't it make way more sense to just do the refund test (one for each currency) where it is the `alpha_ledger`. i.e. Alice funds then Alice refunds. We'd put these tests in a `refund` directory;
- `refund/ethereum_eth.ts`
- `refund/bitcoin_bitcoin.ts`
- `refund/ethereum_erc20`
etc

And then maybe one full refund test (where we also refund `beta_ledger`).

Reply:
Actually, it would.